GNUmakefile

```diff
@@ -109,6 +112,7 @@ deps:  ## Install build and development dependencies
 	go install github.com/golang/protobuf/protoc-gen-go@v1.3.4
 	go install github.com/hashicorp/go-msgpack/codec/codecgen@v1.1.5
 	go install github.com/bufbuild/buf/cmd/buf@v0.36.0
+	go install github.com/hashicorp/go-changelog/cmd/changelog-build@latest
```

Comment:
Ideally https://github.com/hashicorp/go-changelog/ should be tagged with a version we can pin to

Reply:
Agree. Will follow up. I'm also working to optimize the code a bit.